common/script_values/wc_magic_lifestyles_values.txt

```diff
@@ -0,0 +1,545 @@
+﻿wc_magic_bad_1_xp_gain_mult_value = -0.1
+wc_magic_bad_2_xp_gain_mult_value = -0.2
+wc_magic_bad_3_xp_gain_mult_value = -0.3
+wc_magic_good_1_xp_gain_mult_value = 0.1
+wc_magic_good_2_xp_gain_mult_value = 0.2
+wc_magic_good_3_xp_gain_mult_value = 0.3
+wc_magic_bad_noninheritable_xp_gain_mult_value = -0.15
+wc_magic_good_noninheritable_xp_gain_mult_value = 0.15
+
+wc_magic_lifestyle_auto_selection_weight_base_value = 11
+wc_magic_lifestyle_auto_selection_weight_magic_talent_add_to_base_value = {
+	value = 0
+
+	if = {
+		limit = {
+			OR = {
+				has_trait = magic_good
+				has_trait = magic_good_noninheritable
+			}
+		}
+		add = 1989
+	}
+}
+
+wc_magic_lifestyle_auto_selection_weight_education_intrigue_add_to_base_value = 200
+wc_magic_lifestyle_auto_selection_weight_education_martial_add_to_base_value = 500
+wc_magic_lifestyle_auto_selection_weight_education_learning_add_to_base_value = 1000
+
+@wc_local_lifestyle_multiply_mod_positive_big = 10
+@wc_local_lifestyle_multiply_mod_positive_medium = 5
+@wc_local_lifestyle_multiply_mod_positive_small = 3
+@wc_local_lifestyle_multiply_mod_positive_tiny = 1.5
+
+@wc_local_lifestyle_multiply_mod_negative_medium = 0.25
+
+wc_magic_lifestyle_focus_selection_weight_multiply_all_magic_value = { # Common multiplier of all focuses
+	value = 1
+
+	if = {
+		limit = {
+			OR = {
+				has_trait = magic_bad
+				has_trait = magic_bad_noninheritable
+			}
+		}
+		multiply = 0
+	}
+}
+wc_magic_lifestyle_focus_selection_weight_multiply_elemental_spirit_magic_value = {
+	value = 1
+
+	if = {
+		limit = { MAGIC_is_approved_trigger = { MAGIC = elemental_spirit } }
+		multiply = @[wc_local_lifestyle_multiply_mod_positive_big]
+	}
+
+	if = {
+		limit = {
+			MAGIC_is_shunned_or_illegal_trigger = { MAGIC = elemental_spirit }
+			NOT = { has_trait = cynical }
```

Comment:
```suggestion
			NOR = { 
			    has_trait = cynical 
			    has_trait = possessed_1
			    has_trait = possessed_genetic
			    has_trait = lunatic_1
			    has_trait = lunatic_genetic
			    has_trait = ambitious
			}
```
Obviously these are just suggestions and should probably be placed inside a trigger instead.

Reply:
> 
> 
> None of suggested traits imply that character is willing to break religious law easily.
> "Possessed" and "lunatic" as well may represent "Divine madness", like in ck2.
> "Ambitious" only indicates that character want to get something, not that it's ruthless. Actually, "Ambitious" gives +10 zealousness to character personality.

Yeah alright, that makes sense